tests/unary_expression/urnary_expression.js

```diff
@@ -4,3 +4,5 @@ x--;
 -+1;
 x + + + + 1;
 x + (+ (+ (+ 1)))
+x * +y;
++x * y;
```

Comment:
I mean, a \n at the end of the file, to remove the little red `(/)<-` icon on github ui